### PR TITLE
Added nfs support and version command

### PR DESCRIPTION
**Pull request by cableman**
Note: that this requires all are using NFS shares (if docker-compose.yml have it enabled). So this requires information for the whole team before being released

### CODE REVIEW

File: scripts/itkdev-docker-compose [hunk 1]
```diff
@@ -104,6 +105,26 @@ EOF
   fi
 }
 
+function enabledNFS {
+  echo "Setting up nfs..."
+  U=`id -u`
+  G=`id -g`
+
+  LINE="/Users -alldirs -mapall=$U:$G localhost"
```
Comment (rimi-itk) [inline scripts/itkdev-docker-compose:113]: Using `$HOME` rather that `/Users` will probably be more correct here.

File: scripts/itkdev-docker-compose [hunk 2]
```diff
@@ -104,6 +105,26 @@ EOF
   fi
 }
 
+function enabledNFS {
```
Comment (rimi-itk) [inline scripts/itkdev-docker-compose:108]: This function should be called `enableNFS` (without `d`).

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: scripts/itkdev-docker-compose (hunk 1 above)
Comment (cableman) [reply scripts/itkdev-docker-compose:113]: Nope, it should be `/System/Volumes/Data/Users/` for performance and macs fake /Users

File: scripts/itkdev-docker-compose [hunk 3]
```diff
@@ -104,6 +105,26 @@ EOF
...
+  grep -qF -- "$LINE" "$FILE" || sudo echo "$LINE" | sudo tee -a $FILE > /dev/null
+
+  LINE="nfs.server.mount.require_resv_port = 0"
+  FILE=/etc/nfs.conf
+  grep -qF -- "$LINE" "$FILE" || sudo echo "$LINE" | sudo tee -a $FILE > /dev/null
+
+  echo "Restarting nfsd..."
+  sudo nfsd restart
+
+  echo "You should now restarting docker to ensure NFS mounts works..."
```
Comment (turegjorup) [inline scripts/itkdev-docker-compose:125]: *You should now restart docker....

File: scripts/itkdev-docker-compose [hunk 4]
```diff
@@ -125,6 +146,9 @@ Commands:
                 container. Otherwise, the drush container is used to
                 run the command.
 
+  nfs:enable
+                Enabled support for NFS shares (preformance boots on macs)
```
Comment (turegjorup) [inline scripts/itkdev-docker-compose:150]: *Enable

**Review by turegjorup — APPROVED** [review]
Approved with comments

File: scripts/itkdev-docker-compose (hunk 3 above)
Comment (rimi-itk) [reply scripts/itkdev-docker-compose:125]: Can we create a command, `itkdev-compose-compose nfs:check`, say, to check that NFS mounts actually work?

Comment (cableman) [reply scripts/itkdev-docker-compose:125]: Don't known it we could try creating a volume... with the NFS driver:

```
docker volume create --driver local \
    --opt type=nfs \
    --opt o=addr=addr=host.docker.internal,rw,nolock,hard,nointr,nfsvers=3 \
    --opt device=:/Users \
    itkdev-composer-test
```

Use the exit code if `0 ` run `docker volume delete itkdev-composer-test`

Comment (cableman) [reply scripts/itkdev-docker-compose]: Create volume don't fail before the volume is used...

Comment (rimi-itk) [reply scripts/itkdev-docker-compose]: Let's skip this for now.

Comment (cableman) [reply scripts/itkdev-docker-compose]: Nope, this don't work.... not sure why... it works in docker-compose

Comment (cableman) [reply scripts/itkdev-docker-compose]: Mayby `showmount -e | grep /System/Volumes/Data/Users`

File: README.md [hunk 5]
```diff
@@ -20,6 +20,17 @@ docker preferences in the via the dock icon.
 * Disk (Ensure the file has type `raw`)
 * Advanced (CPU's, Memory)
 
+From version 2.x this tool supports the usage of NFS mounted name-volumes, which
+gives faster file synchronisation during composer install and cache clear etc.
+
+It comes with a command `nfs:enable` that exports the users (yours) home directory
```
Comment (rimi-itk) [inline README.md:26]: It doesn't export a home directory (unless you're using your Mac in a very peculiar way), but the `Users` directory.

Maybe just write

> … that exports the `Users` directory through …

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Just a little thing about wording in the documentation.

File: README.md (hunk 5 above)
Comment (cableman) [reply README.md]: Fixed

**Review by rimi-itk — APPROVED** [review]

**Review by turegjorup — APPROVED** [review]

**Pull request merged**